.github/workflows/api-merge.yml

```diff
@@ -0,0 +1,72 @@
+name: API Merge
+
+on:
+  merge_group:
+    types: [checks_requested]
+
+defaults:
+  run:
+    working-directory: api
+
+jobs:
+  test:
+    runs-on: General-Purpose-8c-Runner
+    name: API Unit Tests
+
+    services:
+      postgres:
+        image: postgres:11.12-alpine
```

Comment:
this needs bumping as per https://github.com/Flagsmith/flagsmith/pull/3280

Reply:
Good catch, thank you.